test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -1183,6 +1200,26 @@ Describe "Invoke-WebRequest tests" -Tags "Feature" {
 
     #endregion charset encoding tests
 
+    #region Client Certificate Authentication
+
+    It "Verifies Invoke-WebRequest Certificate Authentication Fails without -Certificate"  {
+        $uri = 'https://localhost:8443/'
+        $result = Invoke-WebRequest -Uri $uri -SkipCertificateCheck
+        $failedMessage = 'Status: Failed'
+        $result.RawContent  | Should Match ([regex]::Escape($failedMessage ))
+    }
+
+    It "Verifies Invoke-WebRequest Certificate Authentication Successful with -Certificate" {
```

Comment:
Maybe `Invoke-WebRequest certificate authentication with -Certificate`.
What IIS or Apache returns in the case? I believe the test shouldn't depend from underlying web server.